bokehjs/src/coffee/core/has_props.coffee

```diff
@@ -388,6 +388,7 @@ export class HasProps extends Backbone.Model
       @document._notify_change(@, attr, old, new_)
 
   materialize_dataspecs: (source) ->
+    # Note: this should be moved to a function separate from HasProps
```

Comment:
It should live on `Glyph`, but since data specs were allowed elsewhere (e.g. in annotations), it would have to be duplicated in `Glyph` and `Renderer`. This way, to avoid duplication, it's either `Model` or `HasProps` (and `Model pretty_much_== `HasProps` in bokehjs). I'm not a big fan of having auxiliary functions accessing internals of classes.